### DISCUSSION TITLE
Rollback removed hub01 basement and NPCs.

### DISCUSSION THREAD

**Discussion by NobleJake** [cataclysmbn/Cataclysm-BN#2807] [General]
### Is your feature request related to a problem? Please describe.

https://github.com/cataclysmbnteam/Cataclysm-BN/issues/1453#issue-1199239412

From the issue, even though the basement of the refugee center is constantly mentioned, but it's still blocked to getting down. It is said that Hub 01 have to also be prevented from entering the basement, just as the refugee center is impossible to enter the basement. However, there is a problem in itself to remove. Hub 01's basement is fully implemented content, unlike the refugee center's, which is not equally comparable. Also, according to the lore, the young children of the merchants of the Free Merchants are staying in the basement of the refugee center, so the game prevents players from going down and killing them, radically different from Hub 01's case.

Hub01 gives the player the choice to cooperate, reject, or invade(3rd choice). Deleting it is equivalent to removing one of the player's options. It is said that the basement of Hub 01 is full of hostile NPCs. Of course, they are loyal security guards of Hub 01, they have a duty to protect the place, and the player is simply an invader, so it is correct that they are hostile. Personally speaking, it seems that they requested deletion of the Hub 01 basement because just hate looting bunch of items, eventhough requires player's high combat skills and near endgame gears to raid.

It is also said that they do not want Hub 01 to be a trivial treasure chest for loot, this does not mean that Hub 01 basement can be deleted for such nonsensible reason. By the time the player decides to loot this place, it will usually be late game, and which is entirely the player's choice and a mechanism driven by curiosity, or just wanted to directly getting through to see what Hub 01's doing. Forcibly blocking the elevator down to the basement doesn't reward the player's curiosity, resulting in a frustrating experience. Therefore, I would like request to return the basement floor of Hub 01 that was originally there.

### Describe the solution you'd like

Rollback the removed hub 01 basement and hostile NPCs.

**Comment by scarf005** [comment]
i think it's a reasonable request. 

### rationale 

- does not interfere with codebase since it's pure JSON changes
- we could port over more hub01 contents from DDA
- it could be done as a single revert commit
